src/script/api/script_list.cpp

```diff
@@ -906,6 +918,7 @@ SQInteger ScriptList::Valuate(HSQUIRRELVM vm)
 				/* See below for explanation. The extra pop is the return value. */
 				sq_pop(vm, nparam + 4);
 
+				vm->_ops_till_suspend_error_threshold = backup_ops_error_threshold;
```

Comment:
Maybe we should make a small Backup wrapper here, so it is automatically restored for us when leaving scope?

Dunno if that is actually an improvement tbh .. but missing a `return` is also an easy mistake made :(

Reply:
Make, or use `AutoRestoreBackup`?